lib/node_modules/@stdlib/iter/for-each/README.md

```diff
@@ -75,20 +75,20 @@ The returned iterator protocol-compliant object has the following properties:
 
 The invoked `function` is provided two arguments:
```

Comment:
```suggestion
The invoked function is provided two arguments:
```

We are not consistent about this across the codebase, but we have been gradually moving away from universally surrounding built-in types with backticks. We might as well address that here.